caravel/utils.py

```diff
@@ -310,6 +310,8 @@ def json_int_dttm_ser(obj):
     val = base_json_conv(obj)
     if val is not None:
         return val
+    if isinstance(obj, date):
+        obj = datetime.combine(obj, datetime.min.time())
     if isinstance(obj, datetime):
```

Comment:
well it's supposed to hit that `else:` if the type is not recognized which it won't now. Also no need to evaluate whether it's a `datetime` if we now its a `date`